pkgs/native_assets_builder/CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+`package:native_assets_builder` and `package:native_assets_cli/native_assets_cli_internal.dart` are used in [`package:dartdev`] and [`package:flutter_tools`].
+
+Dartdev and Flutter Tools are the commandline interfaces for `dart ...` and `flutter ...` commands.
+
+## Downstream builds
+
+Both Dartdev and Flutter Tools are built in open source, and inside g3.
+
+* Dartdev as part of the Dart SDK build on the [Dart CI].
+* Dartdev as part of the g3 build (see CBuild comments on CLs on [Gerrit]).
+* Flutter Tools as part of the workflows on GitHub.
+* Flutter Tools as part of the g3 build (see the "Google testing" workflow on PRs on GitHub).
```

Comment:
Add a link here and in the other places where they are not present yet.

Reply:
I don't know where to link to. I just know it shows up in the GitHub UI on your PR.